src/main/groovy/net/wasdev/wlp/gradle/plugins/utils/ApplicationXmlDocument.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * (C) Copyright IBM Corporation 2017.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.wasdev.wlp.gradle.plugins.utils;
+
+import java.io.File;
+import java.io.IOException;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.TransformerException;
+
+import org.w3c.dom.Element;
+
+public class ApplicationXmlDocument extends XmlDocument {
+    
+    public static final String APP_XML_FILENAME = "install_apps_configuration_1491924271.xml";
```

Comment:
I never did like this number on this file, but it should probably be different than the Maven one.